docs/en/reference/caching.rst

```diff
@@ -395,6 +395,9 @@ cache drivers to manually delete cache entries. For your
 convenience we offer command line tasks to help you with
 clearing the query, result and metadata cache.
 
+**NOTE: It is not possible to use these commands when APC(u)
```

Comment:
```suggestion
**NOTE: It is not possible to use these commands when APCu
```
APCu is now the default, APC has been deprecated for a long time.

Reply:
The problem applies to both, even if APC is deprecated.

Reply:
Oh right, didn't notice we still have docs about APC, my bad.